src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/DefaultMonitorService.java

```diff
@@ -105,9 +116,32 @@ public void releaseResources() {
     MonitorThreadContainer.releaseInstance();
   }
 
+  @Override
+  public synchronized void removeMonitorFromMap(IMonitor monitor) {
```

Comment:
Can this function be just removeMonitor() ?

Does it add something to say it's from a map?